doc/changelog.rst

```diff
@@ -6,6 +6,357 @@ Change Log
+- A ``QobjEvo`` instance is accepted for most operators, e.g.,
+  ``H``, ``c_ops``, ``e_ops``, ``a_ops``.
+- The progress bar is now selected using the ``progress_bar`` option.
+  A new progess bar using the ``tqdm`` Python library is provided.
+- Dynamic arguments, where the value of an operator depends on
+  the current state of the evolution, have been removed. They
+  may be re-implemented later if there is demand for them.
+
+Integrators:
+
+- The SciPy zvode integrator is available with the BDF and
+  Adams methods as ``bdf`` and ``adams``.
+- The SciPy dop853 integrator (an eighth order Runge-Kutta method by
+  Dormand & Prince) is available as ``dop853``.
+- The SciPy lsoda integrator is available as ``lsoda``.
+- QuTiP's own implementation of Verner's "most efficient" Runge-Kutta methods
+  of order 7 and 9 are available as ``vern7`` and ``vern9``. See
+  http://people.math.sfu.ca/~jverner/ for a description of the methods.
+- QuTiP's own implementation of a solver that directly diagonalizes the
+  the system to be integrated is available as ``diag``. It only works on
+  time-independent systems and is slow to setup, but once the diagonalization
+  is complete, it generates solutions very quickly.
+- QuTiP's own implementatoin of an approximate Krylov subspace integrator is
+  available as ``krylov``. This integrator is only usable with ``sesolve``.
+
+Result class:
+
+- A new ``.e_data`` attribute provides expectation values as a dictionary.
+- Both ``.expect`` and ``.e_data`` now return expectation values as lists
+  and not numpy arrays.
+- The contents of the ``.stats`` attribute changed significantly and is
+  now more consistent across solvers.
+
+Monte-Carlo Solver (mcsolve):
+
+- The system, H, may now be a super-operator.
+- The ``seed`` parameter now supports supplying numpy ``SeedSequence`` or
+  ``Generator`` types.
+- The new ``timeout`` and ``target_tol`` parameters allow the solver to exit
+  early if a timeout or target tolerance is reached.
+- The ntraj option no longer supports a list of numbers of trajectories.
+  Instead, just run the solver multiple times and use the class ``MCSolver``
+  if setting up the solver uses a significant amount of time.
+- The ``map_func`` parameter has been replaced by the ``map`` option. In
+  addition to the existing ``serial`` and ``parallel`` values, the value
+  ``loky`` may be supplied to use the loky package to parallelize trajectories.
+- The result returned by ``mcsolve`` now supports calculating photocurrents
+  and calculating the steady state over N trajectories.
+- The old ``parfor`` parallel execution function has been removed from
+  ``qutip.parallel``. Use ``parallel_map`` or ``loky_map`` instead.
+
+Bloch-Redfield Master Eqation Solver (brmesolve):
+
+- The ``a_ops`` and ``spectra`` support implementaitons been heavily reworked to
+  reuse the techniques from the new Coefficient and QobjEvo classes.
+- The ``use_secular`` parameter has been removed.
```

Comment:
Maybe add a comment that ``sec_cutoff=-1'' can be used instead?

Reply:
Fixed!